test/integration/targets/roles_arg_spec/test_path_extension.yml

```
@@ -0,0 +1,32 @@
+- hosts: localhost
+  gather_facts: no
+
+  vars:
+    a_int: 100
+    a_str: "100"
+
+  tasks:
+    - name: Tasks from main
+      import_role:
+          name: a
+          tasks_from: main
+
+    - name: Tasks from main.yml
+      import_role:
+          name: a
+          tasks_from: main
+
+    - name: Tasks from tasks/alternate
+      import_role:
+          name: a
+          tasks_from: taks/alternate
```

```suggestion
          tasks_from: tasks/alternate
```
typo (wondering why this didn't fail the test run though 🤔)

---

That is because we take the `basename()` of the value in `tasks_from`.

https://github.com/ansible/ansible/blob/b9688c5497479fc770143002b1e8e3f3ee5de8df/lib/ansible/playbook/role_include.py#L106-L110